.gitignore

```diff
@@ -18,3 +18,4 @@ dl-cache/*
 !dl-cache/.gitkeep
 qemu-data/
 dcu/
+/logs
```

Comment:
it's looks weird to add absolute path to git ignore, doesn't it?

Reply:
Absolute path notation in gitignore is based on repository's root.
/logs ignores logs directory only within the root, while "logs" would ignore all logs directories, without concert at which depth of a repo it is.

"The pattern hello.* matches any file or directory whose name begins with hello.. If one wants to restrict this only to the directory and not in its subdirectories, one can prepend the pattern with a slash, i.e. /hello.*; the pattern now matches hello.txt, hello.c but not a/hello.java."

Taken from https://git-scm.com/docs/gitignore

Reply:
Thanks for the link. I would argue then `logs` may be better as we do not want to include any logs dirs. But this is fine too I guess.